dist/spec/payid-easy-checkout.txt

```diff
@@ -0,0 +1,728 @@
+     3.1.  PayID Easy Checkout Discovery . . . . . . . . . . . . . .   4
+       3.1.1.  Step 1: Assemble PayID Easy Checkout Discovery URL  .   4
+       3.1.2.  Step 2: Query PayID Easy Checkout Discovery URL . . .   4
+       3.1.3.  Step 3: Parse PayID Easy Checkout Metadata  . . . . .   4
+     3.2.  PayID Easy Checkout URL Assembly  . . . . . . . . . . . .   5
+       3.2.1.  PayID Easy Checkout URL Query Parameters  . . . . . .   5
+   4.  PayID Easy Checkout JRDs  . . . . . . . . . . . . . . . . . .   7
+   5.  Security Considerations . . . . . . . . . . . . . . . . . . .   7
+     5.1.  PayID Easy Checkout Redirection URI Manipulation  . . . .   7
+     5.2.  Access Control  . . . . . . . . . . . . . . . . . . . . .   8
+   6.  IANA Considerations . . . . . . . . . . . . . . . . . . . . .   8
+   7.  Acknowledgments . . . . . . . . . . . . . . . . . . . . . . .   8
+   8.  Appendix  . . . . . . . . . . . . . . . . . . . . . . . . . .   8
+     8.1.  Motivation  . . . . . . . . . . . . . . . . . . . . . . .   8
+       8.1.1.  Design Goals  . . . . . . . . . . . . . . . . . . . .   9
+     8.2.  Example Usage . . . . . . . . . . . . . . . . . . . . . .   9
+       8.2.1.  PayID Easy Checkout Initiation  . . . . . . . . . . .   9
+       8.2.2.  PayID Easy Checkout Wallet Discovery  . . . . . . . .   9
+       8.2.3.  Assemble PayID Easy Checkout URL with Query
+               Parameters  . . . . . . . . . . . . . . . . . . . . .  10
+       8.2.4.  Redirect Payer to Their Wallet  . . . . . . . . . . .  11
+       8.2.5.  Payer Confirms Payment  . . . . . . . . . . . . . . .  11
+   9.  References  . . . . . . . . . . . . . . . . . . . . . . . . .  11
+     9.1.  Normative References  . . . . . . . . . . . . . . . . . .  11
+     9.2.  Informative References  . . . . . . . . . . . . . . . . .  12
+     9.3.  URIs  . . . . . . . . . . . . . . . . . . . . . . . . . .  12
+   Authors' Addresses  . . . . . . . . . . . . . . . . . . . . . . .  12
+
+1.  Terminology
+
+   This protocol can be referred to as the "PayId Easy Checkout
+   Protocol".  It uses the following terminology:
+
+   o  PayID client: a client that queries a PayID server using the PayID
+      Protocol as defined in [PAYID-PROTOCOL][].
+
+
+
+
+Kramer, et al.          Expires January 24, 2021                [Page 2]
+
+Internet-Draft   Draft 1 - PayID Easy Checkout Protocol        July 2020
+
+
+   o  PayID server: the endpoint that returns payment account(s)
+      information, which conforms to the PayID Protocol.
+
+   o  Recipient: Individual or entity receiving a payment (e.g.,
+      e-commerce merchant, charity).
+
+   o  Payer: Individual or entity originating a payment to a
+      "recipient".
+
+   o  Wallet: A device or application that holds funds (may be a non-
+      custodial wallet).
+
+   o  PayID Easy Checkout URL: The URL that is the result of the PayID
+      Easy Checkout protocol; can be used to redirect a client to a
```

Comment:
Define client. Or is it a PayID client?

Reply:
"client" is purposefully left vague throughout the spec.  We don't want to be too prescriptive about how someone could use the `PayID Easy Checkout URL`, thus `client` is just any type of client (ie web client, native app, mobile app etc)

Reply:
If you have a better term that encompasses that, I'm happy to change